### PR TITLE
HBASE-22482 Fix tag Warnings in TestPartitionedMobCompactor

### PR DESCRIPTION
**Pull request by syedmurtazahassan**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 27 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 251 | master passed |
| +1 | compile | 53 | master passed |
| +1 | checkstyle | 69 | master passed |
| +1 | shadedjars | 261 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 168 | master passed |
| +1 | javadoc | 33 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 255 | the patch passed |
| +1 | compile | 64 | the patch passed |
| +1 | javac | 64 | the patch passed |
| +1 | checkstyle | 105 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 408 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1584 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.0.3 3.1.2. |
| +1 | findbugs | 210 | the patch passed |
| +1 | javadoc | 30 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 7968 | hbase-server in the patch passed. |
| +1 | asflicense | 31 | The patch does not generate ASF License warnings. |
| | | 11942 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-255/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/255 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux 1f7d92ce287a 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 73267d9ab1 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-255/2/testReport/ |
| Max. process+thread count | 5247 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-255/2/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 171 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 262 | master passed |
| +1 | compile | 53 | master passed |
| +1 | checkstyle | 75 | master passed |
| +1 | shadedjars | 308 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 252 | master passed |
| +1 | javadoc | 34 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 254 | the patch passed |
| +1 | compile | 51 | the patch passed |
| +1 | javac | 51 | the patch passed |
| +1 | checkstyle | 68 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 262 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1010 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.0.3 3.1.2. |
| +1 | findbugs | 207 | the patch passed |
| +1 | javadoc | 32 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 17076 | hbase-server in the patch failed. |
| +1 | asflicense | 21 | The patch does not generate ASF License warnings. |
| | | 20492 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.quotas.TestQuotaAdmin |
|   | hadoop.hbase.client.TestMobSnapshotCloneIndependence |
|   | hadoop.hbase.replication.multiwal.TestReplicationSyncUpToolWithMultipleWAL |
|   | hadoop.hbase.client.TestFromClientSide |
|   | hadoop.hbase.client.TestFromClientSide3 |
|   | hadoop.hbase.client.TestSnapshotDFSTemporaryDirectory |
|   | hadoop.hbase.util.TestFromClientSide3WoUnsafe |
|   | hadoop.hbase.client.TestAsyncTableAdminApi3 |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-255/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/255 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux 4d6e81b36475 4.4.0-139-generic #165-Ubuntu SMP Wed Oct 24 10:58:50 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 73267d9ab1 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-255/1/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-255/1/testReport/ |
| Max. process+thread count | 5304 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-255/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hbase-server/src/test/java/org/apache/hadoop/hbase/mob/compactions/TestPartitionedMobCompactor.java [hunk 1]
```diff
@@ -811,10 +811,11 @@ private void compareDelFiles(List<CompactionDelPartition> delPartitions) {
...
    * @param basePath the path to create file
-   * @family the family name
-   * @qualifier the column qualifier
-   * @count the store file number
-   * @type the key type
+   * @param family the family name
+   * @param qualifier the column qualifier
+   * @param count the store file number
+   * @param type the key type
+   * @param date the date value
```
Comment (HorizonNet) [inline hbase-server/src/test/java/org/apache/hadoop/hbase/mob/compactions/TestPartitionedMobCompactor.java]: Could you please make it a little bit more specific? What's the purpose of the parameters?

Comment (syedmurtazahassan) [reply hbase-server/src/test/java/org/apache/hadoop/hbase/mob/compactions/TestPartitionedMobCompactor.java]: I have added some more description for parameters. Kinldy check 3rd commit. Thanks.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 26 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 237 | master passed |
| +1 | compile | 53 | master passed |
| +1 | checkstyle | 68 | master passed |
| +1 | shadedjars | 264 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 201 | master passed |
| +1 | javadoc | 33 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 238 | the patch passed |
| +1 | compile | 54 | the patch passed |
| +1 | javac | 54 | the patch passed |
| +1 | checkstyle | 65 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 264 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 728 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | findbugs | 202 | the patch passed |
| +1 | javadoc | 33 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 8712 | hbase-server in the patch passed. |
| +1 | asflicense | 39 | The patch does not generate ASF License warnings. |
| | | 11532 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-255/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/255 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux 5fa6447fe4c1 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 302a9ce056 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-255/3/testReport/ |
| Max. process+thread count | 4767 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-255/3/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**